ksql-engine/src/main/java/io/confluent/ksql/util/KafkaConsumerGroupClientImpl.java

```diff
@@ -67,6 +88,20 @@ public KafkaConsumerGroupClientImpl(KsqlConfig ksqlConfig) {
     return results;
   }
 
+  private File flushPropertiesToTempFile() throws IOException {
+    File configFile = File.createTempFile("ksqlclient", "properties");
+    OutputStreamWriter writer = new FileWriter(configFile);
```

Comment:
I'd avoid `FileWriter` - it uses system default encoding.  How about `FileOutputStream` and `OutputStreamWriter` instead?